src/components/Search/index.tsx

```diff
@@ -203,16 +204,15 @@ function Search({queryJSON, policyIDs, isCustomQuery}: SearchProps) {
         );
```

Comment:
Hmm I'm not sure that we want that for a couple of reasons:

1. Up until this point, we take the type/status from the search results so we can ensure that the data we're rendering is the correct type
2. The doc states that the filters will be dynamic depending on workspace configuration, so we need the search results to be able to render only the relevant filters. This is not implemented in this PR yet, but will soon since the API already returns `statusToShow` in the response

Reply:
Discussing this one with design in Slack

Reply:
I don't think this is a blocker though, we can easily implement it in loading state if we decide to

Reply:
Yea, design is discussing that. We'll see what they come up with 😄